internal/c/libqb.cpp

```diff
@@ -7401,14 +7401,34 @@ void qbg_sub_color(uint32 col1,uint32 col2,uint32 bordercolor,int32 passed){
         return;
     }
 
-    #ifdef QB64_WINDOWS
-        if (write_page->console){
-            HANDLE output = GetStdHandle(STD_OUTPUT_HANDLE);
-            int color = col2 * 16 + col1;
-            SetConsoleTextAttribute(output, color);
-            return;
-        }
-    #endif
+	if (write_page->console){
+		#ifdef QB64_WINDOWS
+			HANDLE output = GetStdHandle(STD_OUTPUT_HANDLE);
+			int color = col2 * 16 + col1;
+			SetConsoleTextAttribute(output, color);
+			return;
+		#else
+		    // accepts colors 0-255 as specified at
+		    // https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit
+			static qbs* ansi; if (!ansi) ansi=qbs_new(0,0);
+			static qbs* closure;
+			if (!closure) {
+				closure=qbs_new(0,0);
+				qbs_set(closure,qbs_new_txt_len(")m",2));

```

Comment:
Are you sure this is the correct sequence? I don't think the escape code should have (parentheses) in it.

Reply:
Yes, there are parenthesis https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit

Reply:
Yes, you misread it :)